### ISSUE TITLE
about the miou and pa?

### ISSUE THREAD

**Issue by louyanyang** [Tramac/Fast-SCNN-pytorch#20]
hi @Tramac , thanks for your job. i have reproduced the miou(54.3%) and pa(92.1%) at cropsize=768 . but i want to know whether you reproduce the paper's result(68.0%) or not.

**Comment by Alohomora0130** [comment]
Hi! Can you share me with the learning rate you use? I try to run the code, but the result is too bad. Could you tell me your learning rate and batch_size? Thank you very much!

**Comment by louyanyang** [comment]
呵呵，我（娄艳阳）已收到您的信，祝你事事顺心，快乐相随
